pkgs/by-name/sw/switch-to-configuration-ng/package.nix

```
@@ -22,6 +24,15 @@ rustPlatform.buildRustPackage {
 
   env.SYSTEMD_DBUS_INTERFACE_DIR = "${buildPackages.systemd}/share/dbus-1/interfaces";
 
+  nativeCheckInputs = [
+    clippy
+  ];
+
+  preCheck = ''
+    echo "Running clippy..."
+    ${lib.getExe cargo} clippy -- -Dwarnings
```

```suggestion
    cargo clippy -- -Dwarnings
```

Bringing in the cargo that will run on the host platform will cause cross-compilation of stc-ng to no longer work.

---

Fixed!